rose/server/score_test.py

```diff
@@ -213,6 +211,7 @@ class TestLimits(SinglePlayerTest):
     """
     Handling movement out of the track
     """
```

Comment:
we can change it to 1 line as the other headers

Reply:
The current style is better and makes it easier to extend the docstring. If we we want to change we better use this style in all docstrings.